src/apps/crud/table/DefaultCRUDTable.jsx

```diff
@@ -98,14 +94,21 @@ const getColDef = (props) => {
   if (props.onView) {
     buttons.push({ icon: RemoveRedEyeOutlinedIcon, onClick: props.onView });
   }
+  if (props.onDownload) {
```

Comment:
Please do not implement non-generic features in the common library. See this: https://github.com/anthony-poon/react.isto-portal-v2/blob/288ee8b7588d6de6db8adee63aef56b0904a1417/src/view/s-admin/admins/SuperAdminListAdminApp.jsx#L98